packages/apps/session-storage/shopify-app-session-storage-prisma/src/prisma.ts

```diff
@@ -129,13 +132,30 @@ export class PrismaSessionStorage<T extends PrismaClient>
     return sessions.map((session) => this.rowToSession(session));
   }
 
+  public async isReady(): Promise<boolean> {
+    try {
+      await this.pollForTable();
```

Comment:
if `this.pollForTable()` catches all errors inside and we're catching all errors here, how do we know if the query succeeded?

Reply:
my bad, I missed the `throw` statement at the end of `pollForTable`